spec/async/http/cache_spec.rb

```diff
@@ -1,6 +1,6 @@
 
 RSpec.describe Async::HTTP::Cache do
 	it "has a version number" do
-		expect(Async::HTTP::Cache::VERSION).not_to be nil
+		expect(Async::HTTP::Cache::VERSION).not_to be_empty
```

Comment:
This seemed like a minor improvement so that a VERSION of `""` wouldn't pass.

Reply:
Actually, I've been rewriting these to be `.to be =~ /\d+\.\d+\.\d+/` or something like that.

Reply:
Makes sense. I'll do that here.

In specs, do you prefer `=~ /.../` to `match(/.../)`?. I've always used the latter and I think it reads well. (And ironically, in our code we've moved away from `=~ /.../` in favor of `.match?(/.../)` based on reading some posts that pointed out that `=~` was slower due to needing to return the full `MatchData` object.)

Reply:
I don’t have a strong opinion either way.